tests/view/writer/writer.js

```diff
@@ -43,6 +43,9 @@ describe( 'Writer', () => {
 
 	describe( 'setSelectionFocus()', () => {
 		it( 'should use selection._setFocus method internally', () => {
+			const position = ViewPosition.createAt( root );
```

Comment:
This seems unrelated but this test started to fail after changing from `before` to `beforeEach`.